packages/core/saved-objects/core-saved-objects-migration-server-internal/src/next.ts

```diff
@@ -257,11 +259,15 @@ export const nextActionMap = (
     MARK_VERSION_INDEX_READY: (state: MarkVersionIndexReady) =>
       Actions.updateAliases({ client, aliasActions: state.versionIndexReadyActions.value }),
     MARK_VERSION_INDEX_READY_SYNC: (state: MarkVersionIndexReady) =>
-      Actions.synchronizeMigrators({
-        waitGroup: updateRelocationAliases,
-        payload: state.versionIndexReadyActions.value,
-        thenHook: (res) => res,
-      }),
+      pipe(
```

Comment:
nit: we could maybe add a comment here explaining that all migrators are trying to update all aliases but only one will succeed